File: 2578-split-with-minimum-sum/2578-split-with-minimum-sum.cpp
class Solution {
public:
    int splitNum(int num) {
        int num1=0;
        int num2=0;
        
        string str=to_string(num);
        sort(str.begin(), str.end());
        
        for(int i=0; i<str.size(); i++)
        {
            if(i%2==0)
            {
                num1=(num1*10)+(str[i]-'0');
            }
            else
            {
                num2=(num2*10)+(str[i]-'0');
            }
        }
        return num1+num2;
    }
};

File: 2578-split-with-minimum-sum/2578-split-with-minimum-sum.java
class Solution {
    public int splitNum(int num) {
        String str=String.valueOf(num);
        char[] digits=str.toCharArray();
        
        Arrays.sort(digits);
        String s1="";
        String s2="";
        
        for(int i=0; i<digits.length; i++)
        {
            if(i%2==0)
            {
                s1+=digits[i];
            }
            else
            {
                s2+=digits[i];
            }
        }
        
        int num1=Integer.valueOf(s1);
        int num2=Integer.valueOf(s2);
        
        return num1+num2;
    }
}